src/packages/documents/documents/components/input-document/input-document.element.ts

```diff
@@ -148,7 +152,7 @@ export class UmbInputDocumentElement extends FormControlMixin(UmbLitElement) {
 
 	#openPicker() {
 		// TODO: Configure the content picker, with `startNodeId` and `ignoreUserStartNodes` [LK]
-		console.log('_openPicker', [this.startNodeId, this.ignoreUserStartNodes]);
+		console.log('#openPicker', [this.startNodeId, this.ignoreUserStartNodes]);
```

Comment:
Do we need to keep the console log?

```suggestion
```

Reply:
I've left the comment in, as it's related to other tasks for wiring up the root (start node) ID and ignore start node features. It's already been removed in another work-in-progress branch.